<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OneLancForm</class>
 <widget class="QWidget" name="OneLancForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>314</width>
    <height>60</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="lbName">
   <property name="geometry">
    <rect>
     <x>62</x>
     <y>8</y>
     <width>190</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btnConn">
   <property name="geometry">
    <rect>
     <x>162</x>
     <y>70</y>
     <width>129</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>连接</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnConf">
   <property name="geometry">
    <rect>
     <x>21</x>
     <y>70</y>
     <width>129</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>设置</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnDisConn">
   <property name="geometry">
    <rect>
     <x>162</x>
     <y>70</y>
     <width>129</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lbConned">
   <property name="geometry">
    <rect>
     <x>225</x>
     <y>19</y>
     <width>55</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QWidget" name="wbg" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>314</width>
     <height>111</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="lbIcon">
   <property name="geometry">
    <rect>
     <x>15</x>
     <y>14</y>
     <width>32</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lbBandWidth">
   <property name="geometry">
    <rect>
     <x>62</x>
     <y>31</y>
     <width>80</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lbPoint">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>25</y>
     <width>10</width>
     <height>10</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <zorder>wbg</zorder>
  <zorder>lbName</zorder>
  <zorder>btnConn</zorder>
  <zorder>btnConf</zorder>
  <zorder>btnDisConn</zorder>
  <zorder>lbConned</zorder>
  <zorder>lbIcon</zorder>
  <zorder>lbBandWidth</zorder>
  <zorder>lbPoint</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
